controllers/api/v1/token_test.go

```diff
@@ -191,6 +193,175 @@ var _ = Describe("TokenController", func() {
 			})
 		})
 	})
+
+	Describe("POST /api/v1/revoke", func() {
+		Context("Given the valid credential", func() {
+			It("returns 204 no content status code", func() {
+				email := "john@example.com"
+				password := "secret"
+				client := FabricateOAuthClient(uuid.New().String(), uuid.New().String())
+				token := FabricateOAuthToken(client)
+				user := FabricateUser("John", email, password)
+
+				form := url.Values{
+					"client_id":     {client.ID},
+					"client_secret": {client.Secret},
+					"token":         {token.GetAccess()},
+				}
+				body := strings.NewReader(form.Encode())
+
+				response := MakeAuthenticatedRequest("POST", "/api/v1/revoke", body, &user)
```

Comment:
@junan Why do we have to make a request with `&user`?

Reply:
Side effect of copy/paste from other test 🙈